.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Run Smoke Tests'
```

Comment:
Neither the scheduled nor the workflow dispatch runs will show up on the PR as one of the associated checks.

We will need some tooling magic to notify folks in their PRs.

A run can be seen here (reachable by visiting the Actions tab):
https://github.com/getsentry/sentry/runs/3414995500?check_suite_focus=true